src/apdu_constants.h

```diff
@@ -145,28 +152,30 @@ void handleSetEth2WinthdrawalIndex(uint8_t p1,
 void handleStarkwareGetPublicKey(uint8_t p1,
                                  uint8_t p2,
                                  const uint8_t *dataBuffer,
-                                 uint16_t dataLength,
+                                 uint8_t dataLength,
                                  unsigned int *flags,
                                  unsigned int *tx);
 void handleStarkwareSignMessage(uint8_t p1,
                                 uint8_t p2,
                                 uint8_t *dataBuffer,
-                                uint16_t dataLength,
+                                uint8_t dataLength,
                                 unsigned int *flags,
                                 unsigned int *tx);
 void handleStarkwareProvideQuantum(uint8_t p1,
                                    uint8_t p2,
                                    const uint8_t *dataBuffer,
-                                   uint16_t dataLength,
+                                   uint8_t dataLength,
                                    unsigned int *flags,
                                    unsigned int *tx);
 void handleStarkwareUnsafeSign(uint8_t p1,
                                uint8_t p2,
                                const uint8_t *dataBuffer,
-                               uint16_t dataLength,
+                               uint8_t dataLength,
                                unsigned int *flags,
                                unsigned int *tx);
 
 #endif
 
+extern uint16_t apdu_response_code;
```

Comment:
Not sure `src/apdu_constants.h` is the bast place for this
Same for the apdu handles btw, maybe split this file in two:
    `src/apdu_constants.h` with the constants
    `src/apdu_handling.h` with the functions prototypes

Reply:
I agree but this would be best looked into by a more in-depth cleanup/refactoring PR, it will do for now. :+1: